tamr_unify_client/client.py

```diff
@@ -145,6 +145,22 @@ def datasets(self):
         """
         return self._datasets
 
+    def create_project(unify, project_config):
```

Comment:
The convention in Python is to use `self` to refer to the object calling this method. Please rename to `unify` to `self`:

```suggestion
    def create_project(self, project_config):
```

and change the code below to use `self` instead of `unify`.